DigitalLearningSolutions.Data.Migrations/202103161535_AddResetPasswordTable.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202103161535)]
+    public class AddResetPasswordTable : Migration
+    {
+        public override void Up()
+        {
+            Create.Table("ResetPassword")
+                .WithColumn("ID").AsInt32().NotNullable().PrimaryKey().Identity()
+                .WithColumn("ResetPasswordHash").AsCustom("text").NotNullable()
+                .WithColumn("PasswordResetDateTime").AsCustom("timestamp").NotNullable();
+
+            Alter.Table("AdminUsers")
+                .AddColumn("ResetPasswordID").AsInt32().Nullable();
+
+            Alter.Table("Candidates")
+                .AddColumn("ResetPasswordID").AsInt32().Nullable();
```

Comment:
Ah, I think these should be marked as foreign keys (which means the order of deletions should be different too)